celery/canvas.py

```diff
@@ -530,8 +523,6 @@ def _apply_async(self):
     kwargs = getitem_property('kwargs', 'Keyword arguments to task.')
     options = getitem_property('options', 'Task execution options.')
     subtask_type = getitem_property('subtask_type', 'Type of signature')
-    chord_size = getitem_property(
```

Comment:
are we spliting chord_size with chord_length & chord_size_bytes?

Reply:
Removed the references to `chord_length` to avoid confusion.  Here `chord_size` is removed from the chord object since it is set on the redis backend instead.  

`chord_size_bytes` exists because redis returns a integer back as bytes.
See https://github.com/andymccurdy/redis-py/blob/1a41cfd95a53a95b078084d8627be6b6fba3bb71/tests/test_commands.py#L219

Reply:
cool thanks for clearing